### PR TITLE
move test command to gcb

### PR DESCRIPTION
**Pull request by rkpagadala**
move release qualification test command from airflow to google cloud builder

### CODE REVIEW

File: release/airflow/airflow_scripts.sh [hunk 1]
```diff
@@ -23,6 +23,7 @@ SCRIPTPATH=$( pwd -P )
 # shellcheck source=release/airflow/gcb_build_lib.sh
 source "${SCRIPTPATH}/gcb_build_lib.sh"
 
+# this function is a helper function and not called directly by airflow
```
Comment (hklai) [inline release/airflow/airflow_scripts.sh]: Probably make these more compact?
E.g.
```
# Helper. Not called directly by Airflow.
# Called directly by Airflow.
```

File: release/airflow/airflow_scripts.sh [hunk 2]
```diff
@@ -97,23 +101,23 @@ function build_template() {
...
 }
 
+# this function is called directly by airflow
 function test_command() {
-    cp /home/airflow/gcs/data/githubctl ./githubctl
-    chmod u+x ./githubctl
-    pwd; ls -l    ./githubctl
+    local KEY_FILE_PATH
+    local WAIT_FOR_RESULT
+    KEY_FILE_PATH=""
```
Comment (hklai) [inline release/airflow/airflow_scripts.sh]: why is this always empty?

File: release/test_release.sh [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+[[ -z "${DOCKER_HUB}"     ]] && usage
+[[ -z "${GCS_BUILD_PATH}" ]] && usage
+[[ -z "${VERSION}"        ]] && usage
+
+
+githubctl_setup
+github_keys
+
+git config --global user.name "TestRunnerBot"
+git config --global user.email "testrunner@istio.io"
```
Comment (hklai) [inline release/test_release.sh]: why are these needed for running githubctl?

File: release/airflow/airflow_scripts.sh (hunk 2 above)
Comment (rkpagadala) [reply release/airflow/airflow_scripts.sh]: this is there if we want to override the path in development. We dont use this when the pipeline is actually running

Comment (hklai) [reply release/airflow/airflow_scripts.sh]: Why is it passed to the runbuild comand below if it is not used in development?

**Comment by rkpagadala** [comment]
@hklai fixed

File: release/airflow/airflow_scripts.sh [hunk 4]
```diff
@@ -61,28 +63,23 @@ function create_subs_file() {
...
+         "${SUBS_FILE}" "${PROJECT_ID}" "${SVC_ACCT}" "${WAIT_FOR_RESULT}"
     exit "${BUILD_FAILED}"
 }
 
+# Called directly by Airflow.
 function build_template() {
-    local KEY_FILE_PATH
     local WAIT_FOR_RESULT
-    KEY_FILE_PATH=""
     WAIT_FOR_RESULT="true"
```
Comment (hklai) [inline release/airflow/airflow_scripts.sh]: Similarly, is WAIT_FOR_RESULT needed? It seems to be always true?

File: release/generate_manifest.sh [hunk 5]
```diff
@@ -318,6 +292,8 @@ pushd "${CLONE_DIR}"
   #copy the needed files
   gsutil cp "${MANIFEST_FILE}" "${BASE_MANIFEST_URL}"
   gsutil cp "${MANIFEST_FILE}" "${MANIFEST_URL}"
+  gsutil cp "${MANIFEST_FILE}" "${GCS_BUILD_PATH}"
```
Comment (hklai) [inline release/generate_manifest.sh]: Why is the manifest file needed in different places?

And why
  BASE_MANIFEST_URL="gs://istio-release-pipeline-data/release-tools/${BRANCH}-manifest.xml"
  BASE_MASTER_MANIFEST_URL="gs://istio-release-pipeline-data/release-tools/master-manifest.xml"

**Comment by rkpagadala** [comment]
@hklai PTAL

**Comment by hklai** [comment]
@rkpagadala There are still two copies as in https://github.com/istio/istio/blob/fb861e733b89224430323e78138e830d37a1dae3/release/generate_manifest.sh#L291

Why is that needed?

**Comment by rkpagadala** [comment]
one is used by subsequent steps in the build, the other one is used to compare and make sure that subsequent builds are from shas after the current one

**Comment by hklai** [comment]
How are they used? And why is one of them specific to master?
